dandischema/consts.py

```diff
@@ -1,5 +1,5 @@
-DANDI_SCHEMA_VERSION = "0.4.4"
-ALLOWED_INPUT_SCHEMAS = ["0.3.0", "0.3.1", "0.4.0", "0.4.1", "0.4.2", "0.4.3"]
+DANDI_SCHEMA_VERSION = "0.4.5"
+ALLOWED_INPUT_SCHEMAS = ["0.3.0", "0.3.1", "0.4.0", "0.4.1", "0.4.2", "0.4.3", "0.4.4"]
```

Comment:
making model more restricting by mandating some fields whenever they were optional before, might better come with `.minor` version boost, not just `.patch` since means that prior legit models might not be compatible with this newer one.
```suggestion
DANDI_SCHEMA_VERSION = "0.5.0"
ALLOWED_INPUT_SCHEMAS = ["0.3.0", "0.3.1", "0.4.0", "0.4.1", "0.4.2", "0.4.3", "0.4.4"]
```
?